docs/backup-restore.md

```diff
@@ -0,0 +1,106 @@
+# Scalar DL Backup Creation and Restoration
+
+Since Scalar DL provides transaction capability on top of non-transactional (possibly transactional) databases non-invasively, you need to take special care of backing up and restoring the databases in a transactionally-consistent way.
+This guide shows you how to create and restore transactionally-consistent Scalar DL backups.
+
+## How to create transactionally consistent backup 
+
+Scalar DL service is integrated with the admin interface, which allows you to pause the Scalar DL services using the scalar-admin client tool.
```

Comment:
It doesn't explain how to create backups. (so not consistent with what the title says)
It only explains the following things.
* Scalar DL is integrated with the admin interface 
* scalar-admin client tool can pause Scalar DL

Reply:
Sorry for the inconvenience.
Document updated with new structure and sentences.